### PR TITLE
#162948865 Redux SetUp

### PR DESCRIPTION
**Pull request by Deekerubo**
#### What does this PR do?
* Setup react redux

#### Description of Task to be completed?
* Setup react redux

#### How should this be manually tested?
* After cloning the repository
cd into  `ch-redux-setup-162948865`
* Install [Redux Devtools](https://github.com/zalmoxisus/redux-devtools-extension) chrome extension in your browser
* Run `npm install` to install all the modules listed in the `package.json` file.
* Run `npm start` command to start the application.
It should bring a page displayed below:
screenshot 2019-02-07 at 22 17 18

* Click on the `Redux DevTools` extension on the Chrome browser and it should display the store as shown below.
screenshot 2019-02-07 at 22 17 59

#### What are the relevant pivotal tracker stories?
[#162948865](https://www.pivotaltracker.com/story/show/162948865)

### CODE REVIEW

**Comment by danuluma** [comment]
Please fix the merge conflicts

**Comment by boltonotieno** [comment]
You have seven commits, kindly squash them into 1.

**Review by Ruiru11 — APPROVED** [review]
good work. kindly implement Bolton's feedback.

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
 import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
+import store from "./redux/store";
 
-ReactDOM.render(<App />, document.getElementById("root"));
+ReactDOM.render(
```
Comment (Betty-Kebenei) [inline src/index.js:7]: This implementation will later cause problems. Kindly do the routing in the App.jsx file. This file should just remain as before.

Comment (Betty-Kebenei) [inline src/index.js:7]: This implementation will later cause problems. Kindly do the routing in the App.jsx file. This file should just remain as before.

**Review by Betty-Kebenei — COMMENTED** [review]
look at this `https://github.com/andela/ah-alpha-frontend/pull/2/files`

**Pull request merged**